tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/ScopedBuilder.scala

```diff
@@ -219,4 +219,11 @@ class ScopedBuilder
     funDef(e, tuple(elems: _*), times(sets: _*))
   }
 
+  /**
+   * A name expression referring to the TlaDecl
+   */
+  def declAsNameEx(decl: TlaDecl): TBuilderInstruction = {
```

Comment:
Just a note, you don't want this method for all `TlaDecl`, in particular not for `TlaOperDecl`. If you're referencing declarations, you'd need `appDecl` (see current builder, not ported yet), because it's easy to accidentally write `A`, where you needed to have a `A()`.